packages/design-system/src/components/Icon/IconSVG.tsx

```diff
@@ -168,19 +168,13 @@ IconSVG.Info = () => (
   </BaseSVG>
 );
 
-IconSVG.Info = () => (
```

Comment:
Can we keep the info icon?

Reply:
There's another (identical) `Info` icon right above it!